### PR TITLE
Add Redux framework

### PR DESCRIPTION
**Pull request by iuricmp**
This pull request adds the Redux framework to the project, including the necessary files and dependencies.

### CODE REVIEW

File: mobile/context/auth/index.tsx [hunk 1]
```diff
@@ -53,17 +48,18 @@ function useProtectedRoute(user: User | null) {
...
     <AuthContext.Provider
-      value={{
-        signIn: (user) => setAuth(user),
-        signOut: () => setAuth(null),
-        user,
-      }}
+      value={
+        {
+          // signIn: (user) => setAuth(user),
+          // signOut: () => setAuth(null),
```
Comment (jefft0) [inline mobile/context/auth/index.tsx:60]: This is commented out. Should it be removed? Or if it should be kept, maybe add a comment to explain?

**Comment by jefft0** [comment]
In a fresh clone of the branch, I do `cd gnosocial/mobile` and `make ios` . When the app starts it prints:
`Unable to resolve "@gno/redux/store" from "app/(root)/settings.tsx"`

File: mobile/app/(auth)/import-account.tsx [hunk 2]
```diff
@@ -4,10 +4,11 @@ import TextInput from "components/textinput";
 import Button from "components/button";
 import Spacer from "components/spacer";
 import { useGno } from "@gno/hooks/use-gno";
-import { useAuth } from "context/auth";
 import SeedBox from "components/seedbox";
 import { ModalConfirm } from "components/modal";
 import Alert from "components/alert";
+import { logedIn } from "redux/features/accountSlice";
```
Comment (jefft0) [inline mobile/app/(auth)/import-account.tsx:10]: "logedIn" should be "loggedIn"

File: mobile/context/auth/index.tsx (hunk 1 above)
Comment (iuricmp) [reply mobile/context/auth/index.tsx]: It was removed in the latest commit. Thanks for the notice.

**Comment by iuricmp** [comment]
`make ios` must work now.

**Review by jefft0 — APPROVED** [review]
Builds and runs on Android and iOS simulators.

**Pull request merged**